force-app/main/default/classes/RD2_RecurringDonations_TDTM.cls

```diff
@@ -108,6 +108,7 @@ public class RD2_RecurringDonations_TDTM extends TDTM_Runnable {
         List<SObject> rds, List<SObject> oldRds,
         TDTM_Runnable.Action triggerAction, Schema.DescribeSObjectResult objResult
     ) {
+        TDTM_TriggerHandler.enableOldListCachingEvaluation(triggerAction, rds?.size());
```

Comment:
I feel like TDTM_TriggerHandler should already know the current triggerAction / size. Is there a way for us to not pass these two variables back up to it?